src/geometry/euclidean/circumcenter.lean

```diff
@@ -335,6 +335,79 @@ begin
   exact h.2
 end
 
+/-- The circumradius is non-negative. -/
+lemma circumradius_nonneg {n : ℕ} (s : simplex ℝ P n) : 0 ≤ s.circumradius :=
+s.dist_circumcenter_eq_circumradius 0 ▸ dist_nonneg
+
+/-- The circumradius of a simplex with at least two points is
+positive. -/
+lemma circumradius_pos {n : ℕ} (s : simplex ℝ P (n + 1)) : 0 < s.circumradius :=
+begin
+  refine lt_of_le_of_ne s.circumradius_nonneg _,
+  intro h,
+  have hr := s.dist_circumcenter_eq_circumradius,
+  simp_rw [←h, dist_eq_zero] at hr,
+  have h01 := (injective_of_affine_independent s.independent).ne
+    (dec_trivial : (0 : fin (n + 2)) ≠ 1),
+  simpa [hr] using h01
+end
+
+/-- The circumcenter of a 0-simplex equals its unique point. -/
+lemma circumcenter_eq_point (s : simplex ℝ P 0) (i : fin 1) :
+  s.circumcenter = s.points i :=
+begin
+  have h := s.circumcenter_mem_affine_span,
+  rw [set.range_unique, mem_affine_span_singleton] at h,
+  rw h,
+  congr
+end
+
+/-- The circumcenter of a 1-simplex equals its centroid. -/
+lemma circumcenter_eq_centroid (s : simplex ℝ P 1) :
+  s.circumcenter = finset.univ.centroid ℝ s.points :=
```

Comment:
Idem dito?

Reply:
It's arguable that the centroid is simpler than the circumcenter, even if the expression is longer, but I think it depends very much on what you want to do with the point afterwards; it's not clear either expression is going to be useful much more often. (Unlike results such as `dist_circumcenter_eq_circumradius`, which I think does clearly convert a more complicated expression into a simpler one.)

Reply:
ok